### PR TITLE
Fix error when closing the alert with an error submitting the form, cleared fields

### PR DESCRIPTION
**Pull request by tavriaforever**
// cc @gela-d

### CODE REVIEW

File: desktop.blocks/comment/comment.js [hunk 1]
```diff
@@ -164,8 +125,9 @@ modules.define('comment', ['i-bem__dom', 'jquery'], function (provide, BEMDOM, $
                     context: this
                 }).done(function () {
                     this.emit('comment:delete');
-
                     BEMDOM.destruct(this.domElem);
+                }).fail(function () {
+                    alert('Не удалось удалить комментарий');
```
Comment (tadatuta) [inline desktop.blocks/comment/comment.js:130]: we've got great `modal` block ;)

File: desktop.blocks/forum-form/forum-form.js [hunk 2]
```diff
@@ -217,6 +214,28 @@ modules.define('forum-form', ['i-bem__dom', 'jquery'], function (provide, BEMDOM
...
+
+        /**
+         * Hide form`s processing,
+         * if stop processing call after error,
+         * form will not cleared
+         * @param isError
+         * @returns {Object} - this block
+         */
+        hideProcessing: function (isError) {
+            !isError && this._clearForm();
```
Comment (tadatuta) [inline desktop.blocks/forum-form/forum-form.js:235]: `isError || this._clearForm();` is one symbol shorter

File: desktop.blocks/forum-form/forum-form.js [hunk 3]
```diff
@@ -217,6 +214,28 @@ modules.define('forum-form', ['i-bem__dom', 'jquery'], function (provide, BEMDOM
...
+        },
+
+        /**
+         * Show form`s processing
+         * @returns {Object} - this block
+         */
+        showProcessing: function () {
+            this.setMod('processing', 'yes');
+            return this;
+        },
```
Comment (gela-d) [inline desktop.blocks/forum-form/forum-form.js:225]: Maybe write here `return this.setMod('processing', 'yes');`?

File: desktop.blocks/comment/comment.js (hunk 1 above)
Comment (tavriaforever) [inline desktop.blocks/comment/comment.js:130]: I have a task in which I plan to replace everywhere on modal

File: desktop.blocks/forum-form/forum-form.js (hunk 2 above)
Comment (tavriaforever) [inline desktop.blocks/forum-form/forum-form.js]: `isNotError`? )

File: desktop.blocks/forum-form/forum-form.js (hunk 3 above)
Comment (tavriaforever) [inline desktop.blocks/forum-form/forum-form.js:225]: Yes, you are right

File: desktop.blocks/forum-form/forum-form.js (hunk 2 above)
Comment (tavriaforever) [inline desktop.blocks/forum-form/forum-form.js:235]: isError || this._clearForm() required use `false` in arguments, so the count of symbols would be the same

Comment (tadatuta) [inline desktop.blocks/forum-form/forum-form.js:235]: @tavriaforever I don't care much about symbols but I can't understand why should something be changed in arguments (as there `&&` -> `||` in code)

**Pull request merged**